### PR TITLE
Documentation(1046260): Move getting started documentation into inputs for instant publish

### PR DESCRIPTION
**Pull request by Mohamed7Imran**
### Feature description
Move getting started documentation into inputs for instant publish.

### Code Studio usage(Mandatory)
* Code Studio used in this PR/MR?
    - [ ] Yes
    - [x] No
* If `Yes`: Primary use (choose one)
    - [ ] Generate new code
    - [ ] Refactor/improve existing code
    - [ ] Tests
    - [ ] Bug fix / debugging help
    - [ ] Docs / comments
    - [ ] Review assistance (explanations/summaries)
    - [ ] Other: 

* MD File

* Outcome
    - [ ] Saved time
    - [ ] Neutral
    - [ ] Cost time
* If “Cost time” explain in short (1 or 2 lines):

### Requirement and specification document
* [ ] API Review (Task link: _________________)
* [x] Azure Task with detailed feature information (Task link: https://dev.azure.com/EssentialStudio/Ej2-Web/_workitems/edit/1041955)
* [ ] Not applicable

### Feature scope and impact
* [ ] Core functionality (high impact)
* [x] Enhancement to existing feature
* [ ] New standalone feature
* [ ] Experimental feature

### Feature matrix documentation
* [ ] Feature matrix document updated [Centralized location](https://syncfusion.sharepoint.com/:f:/r/sites/Blazor/Shared%20Documents/Feature%20Matrix%20-%20Documents?csf=1&web=1&e=OGX1td)
* [x] Not applicable

### Testing coverage
* [ ] Tested against feature matrix
* [x] Edge cases and error scenarios verified
* [ ] Cross-browser testing completed

### Test cases and automation
* [ ] Test cases documented and attached to this PR
* [ ] BUnit tests added (PR link: _________________)
* [ ] Playwright tests added (PR link: _________________)
* [ ] Manual testing only (justification: _________________)
* [x] Not applicable

### Test bed sample location
Provide the test bed sample location for code reviewers to verify the feature's behavior.
 
### Platform verification
* [ ] Verified in Blazor Server
* [ ] Verified in Blazor WASM
* [x] Not applicable

### Related components
* [ ] Feature applicable to other components (Tasks created: _________________) (tagged "needs-attention-coreteam")
* [x] Not applicable to other components

### Breaking changes
* [ ] Contains breaking changes - provide migration guidance (added "breaking-change" label)
* [x] No breaking changes

### Performance verification
* [ ] Memory usage verified (no leaks)
* [ ] Rendering performance verified
* [x] Not applicable

### Reviewer Checklist
* [ ] Feature matrix documentation verified
* [ ] Test coverage verified
* [x] Sample implementation reviewed
* [x] Code meets standards and best practices
* [ ] C# implementation used appropriately (minimal JavaScript)

### CODE REVIEW

File: ej2-asp-core-mvc/form-renderer/EJ2_ASP.MVC/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+{% endtabs %}
+
+Press <kbd>Ctrl</kbd>+<kbd>F5</kbd> (Windows) or <kbd>⌘</kbd>+<kbd>F5</kbd> (macOS) to run the app. Then, the Syncfusion<sup style="font-size:70%">&reg;</sup> ASP.NET MVC Form Renderer control will be rendered in the default web browser.
+
+The output looks like below
+
+![ASP.NET MVC Form Renderer](../images/form-renderer.png)
+
+
+## See also
```
Comment (sabari-senthamaraikannan) [inline ej2-asp-core-mvc/form-renderer/EJ2_ASP.MVC/getting-started.md]: Remove the See also content

File: ej2-asp-core-mvc/form-renderer/EJ2_ASP.NETCORE/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+{% endhighlight %}
+{% endtabs %}
+
+## Run the application
+
+Press <kbd>Ctrl</kbd>+<kbd>F5</kbd> (Windows) or <kbd>⌘</kbd>+<kbd>F5</kbd> (macOS) to launch the application. The ASP.NET Core Form renderer will render in your default web browser.
+
+![ASP.NET Core Form Renderer](../images/form-renderer.png)
+
+## See also
```
Comment (sabari-senthamaraikannan) [inline ej2-asp-core-mvc/form-renderer/EJ2_ASP.NETCORE/getting-started.md]: Remove the See also content

**Review by sabari-senthamaraikannan — APPROVED** [review]
